src/stores/index.js

```diff
@@ -108,6 +113,9 @@ class Store {
         }
       ],
       proposals: [
+      ],
+      leaderboard: [
+
```

Comment:
do we need this empty line?
can you please make this in 1 one line like this?
```
proposals: [],
leaderboard: [],
```

Reply:
FIXED